seeding.py

```diff
@@ -60,29 +69,40 @@ def seed_events(db, seed_path):
         db_event = db.events.find_one({'eid': event["eid"]})
         if db_event:
             continue
+
         event["participants"] = []
         for p in db["members"].find({}):
+            p['food'] = random.choices(bool_list, weights=[0.8, 0.2])[0]
+            p['transport'] = random.choice(bool_list)
+            p['dietaryRestrictions'] = random.choices(
+                dietaryRestriction_list, weights=[0.9, 0.05, 0.03, 0.02])[0]
+            x = random.choices(
```

Comment:
What is happening here? :smile:

Reply:
Ugly code, will cleanup. Is for distributing participant data in a real life scenario to verify charts/graphs act as they should.